projects/vorbis/decode_fuzzer.cc

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
```

Comment:
We encourage to have the fuzz targets (this file) in the project's repository, not here. 
https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md

Again, this doesn't have to happen in the first commit (but still welcome)

Reply:
The plan is to upstream all of this once it's running, as @reaperhulk and I have done on a few other targets :-)